packages/maskbook/src/plugins/Trader/settings.ts

```diff
@@ -52,7 +52,7 @@ export const polygonNetworkTradeProviderSettings = createGlobalSettings<TradePro
     { primary: () => '' },
 )
 
-export const bscNetworkTradeProviderSettings = createGlobalSettings<TradeProvider>(
+export const BinanceNetworkTradeProviderSettings = createGlobalSettings<TradeProvider>(
```

Comment:
`binanceNetworkTradeProviderSettings` would be better. Keep the same style with `ethereumNetworkProviderSettings`.

Reply:
@guanbinrui